<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn1">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>12</string>
        </property>
        <property name="primary" stdset="0">
         <string/>
        </property>
        <property name="type" stdset="0">
         <string>primary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn2">
        <property name="text">
         <string>26</string>
        </property>
        <property name="type" stdset="0">
         <string>primary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn3">
        <property name="text">
         <string>31</string>
        </property>
        <property name="type" stdset="0">
         <string>primary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn4">
        <property name="text">
         <string>66</string>
        </property>
        <property name="type" stdset="0">
         <string>primary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn5">
        <property name="text">
         <string>3</string>
        </property>
        <property name="type" stdset="0">
         <string>primary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPB">
        <property name="text">
         <string>2</string>
        </property>
        <property name="type" stdset="0">
         <string>danger</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="btnSubmit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Get Rich!</string>
      </property>
      <property name="type" stdset="0">
       <string>submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
